Runtime/Native/Windows/NativeClient.cs

```diff
@@ -0,0 +1,438 @@
+                    string.Format("Temp/{0}/{1}/crashes", Application.companyName, Application.productName));
+
+            if (string.IsNullOrEmpty(nativeCrashesDir) || !Directory.Exists(nativeCrashesDir))
+            {
+                yield break;
+            }
+            var crashDirs = Directory.GetDirectories(nativeCrashesDir);
+            foreach (var crashDir in crashDirs)
+            {
+
+                var crashDirFullPath = Path.Combine(nativeCrashesDir, crashDir);
+                var crashFiles = Directory.GetFiles(crashDirFullPath);
+
+                var alreadyUploaded = crashFiles.Any(n => n.EndsWith("backtrace.json"));
+                if (alreadyUploaded)
+                {
+                    continue;
+                }
+                var minidumpPath = crashFiles.FirstOrDefault(n => n.EndsWith("crash.dmp"));
+                if (string.IsNullOrEmpty(minidumpPath))
+                {
+                    continue;
+                }
+                if (attachments == null)
+                {
+                    attachments = new List<string>();
+                }
+                attachments.Concat(crashFiles.Where(n => n != minidumpPath));
+                IDictionary<string, string> attributes = GetScopedAttributes();
+                // be sure that error.type attribute provided by default by our library
+                // is always present in native attributes.
+                attributes["error.type"] = "Crash";
+
+                yield return backtraceApi.SendMinidump(minidumpPath, attachments, attributes, (BacktraceResult result) =>
+                {
+                    if (result != null && result.Status == BacktraceResultStatus.Ok)
+                    {
+                        File.Create(Path.Combine(crashDirFullPath, "backtrace.json"));
+                    }
+                });
+            }
+        }
+        /// <summary>
+        /// Generate path to Crashpad handler binary
+        /// </summary>
+        /// <returns>Path to crashpad handler binary</returns>
+        private string GetDefaultPathToCrashpadHandler()
+        {
+            const string crashpadHandlerName = "crashpad_handler.dll";
+            const string pluginDir = "Plugins";
+            string architecture = IntPtr.Size == 8 ? "x86_64" : "x86";
+
+            string pluginPath = Path.Combine(pluginDir, architecture);
+            string pluginHandlerPath = Path.Combine(pluginPath, crashpadHandlerName);
+
+            // generate full path to .dll file in plugins dir.
+            return Path.Combine(Application.dataPath, pluginHandlerPath);
+
+        }
+        /// <summary>
+        /// Clean scoped attributes
+        /// </summary>
+        internal static void CleanScopedAttributes()
+        {
+            // cleaning scoped attributes should be skipped when 
+            // Configuration.SendUnhandledGameCrashesOnGameStartup  is set to false
```

Comment:
I see we always `CleanScopedAttributes` when we construct `NativeClient`. I think we don't check `Configuration.SendUnhandledGameCrashesOnGameStartup` before we clean scoped attributes.

Reply:
We're reading them to memory before we start yielding.

Reply:
I think we need to do the same thing we do with Breadcrumbs file now